packages/core/src/components/FacetSearch/FacetSearch.stories.tsx

```diff
@@ -0,0 +1,22 @@
+import { Meta, StoryObj } from "@storybook/react";
+import FacetSearchRaw from "./FacetSearch?raw";
+import { FacetSearch } from "./FacetSearch";
+
+const meta: Meta = {
```

Comment:
The docs page looks a bit strange without the component props table... but I don't have any good alternative 😐 
We could make this a sample of another component, but which one? Accordion? List? CheckboxGroup?

Reply:
I agree, the Accordion is the one that makes the most sense to me.

Reply:
Yes, I think adding this sample to the Accordion page seems to make sense to me as well